requirements/base.txt

```diff
@@ -381,6 +389,8 @@ packaging==23.0
     # via djangocms-text-ckeditor
 phonenumbers==8.12.33
     # via -r requirements/base.in
```

Comment:
you should be able to drop this as phonenumberslite is (arbitrarily) choosen in maykin-2fa

Reply:
We have custom validators using `phonenumbers`; sticking with this for now.

Reply:
we may need to make this a configurable group then in maykin-2fa since that pulls phonenumberslite and they may both install their code to the `phonenumbers` module :thinking: so that could cause weird issues

do you get any warnings from pip if you install everything?

Reply:
No warnings on my end

Reply:
sounds good, if you run into issues after all in the end, we'll make the necessary changes in maykin-2fa :+1: